Lets Do This/Controllers/Login/LDTUserRegisterViewController.m

```diff
@@ -181,6 +182,10 @@ - (IBAction)submitButtonTouchUpInside:(id)sender {
             }];
 
         } failure:^(NSError *error) {
+            // Only record error in Crashlytics if error is NOT lack of connectivity, timeout, or email already exists.
+            if ((error.code != -1009) && (error.code != -1001) && (error.code != 422)) {
```

Comment:
Refs #903 -- don't think we need to log any "email already exists" errors as non-fatal, where as 401 sounds like a more  important one to log.